lib/services/database/post_repository_service.dart

```diff
@@ -31,7 +43,13 @@ class PostRepositoryService {
 
   /// This method deletes the post with id [postId] from the database
```

Comment:
I think it is worht mentionning it also removes all its subcollection:
```suggestion
  /// This method deletes the post with id [postId] from the database.
  /// This means removing the corresponding document, and all the
  /// subcollections.
```

Reply:
8098704de70971a2826a70ea6e0b88afc73ee568